### ISSUE TITLE
Do you support Chinese Language?

### ISSUE THREAD

**Issue by ken30huang** [denser-org/denser-chat#3]
I want to use read Chinese Documents，these Documents are come from company inside。
I want to build a little AI app，in order to we can chat with these document。

By the way , we can use ChatGPT or Claude API ， but I am use other api.

If I want to DenserAI to build a AI app like this , how can I do this ？

**Comment by zhiheng-huang** [comment]
Yes. The code supports Chinese. You can follow the README.md to build a chatbot on your own data. Specifically, edit the sources.txt (https://github.com/denser-org/denser-chat/blob/main/denser_chat/sources.txt) file to point to your documents.
